### ISSUE TITLE
Simple AND / OR conditions should be placed on same line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#997]
We should report an issue in case a "simple" binary condition spans multiple lines.

Following condition should trigger:
```c#
public object SomeProperty { get; set; }

public void DoSomething(object o)
{
    if (SomeProperty != null
        && SomeProperty.Equals(o))
    { }
}
```
It should be fixed to
```c#
public object SomeProperty { get; set; }

public void DoSomething(object o)
{
    if (SomeProperty != null && SomeProperty.Equals(o))
    { }
}
```

Same is valid for some more complex parts:
```c#
public object SomeProperty1 { get; set; }

public object SomeProperty2 { get; set; }

public void DoSomething(object o)
{
    if ((SomeProperty1 != null
        && SomeProperty1.Equals(o))
     || (SomeProperty2 != null
        && SomeProperty2.Equals(o)))
    { }
}
```
It should be fixed to something like
```c#
public object SomeProperty1 { get; set; }

public object SomeProperty2 { get; set; }

public void DoSomething(object o)
{
    if ((SomeProperty1 != null && SomeProperty1.Equals(o))
     || (SomeProperty2 != null && SomeProperty2.Equals(o)))
    { }
}

```
